src/azul/service/responseobjects/step_function_helper.py

```diff
@@ -0,0 +1,59 @@
+import json
+
+from azul.deployment import aws
+
+
+class StepFunctionHelper:
+    """
+    Wrapper around boto3 SFN client to handle resource name generation and state machine executions
+    """
+    def state_machine_arn(self, state_machine_name):
+        return f'arn:aws:states:{aws.region_name}:{aws.account}:stateMachine:{state_machine_name}'
+
+    def execution_arn(self, state_machine_name, execution_name):
+        return f'arn:aws:states:{aws.region_name}:{aws.account}:execution:{state_machine_name}:{execution_name}'
+
+    def start_execution(self, state_machine_name, execution_name=None, execution_input=None):
```

Comment:
Correct me if I am wrong but we don't use the cases were execution name and input are optional. Let's make those parameters mandatory. That would be adding to the value of this class.

Reply:
We don't use the case where execution_input is None but right now execution_name is always not given because aws will generate a uuid for us